code/modules/admin/verbs/playsound.dm

```diff
@@ -138,6 +138,8 @@
 					to_chat(world, span_boldannounce("An admin played: [webpage_url]"), confidential = TRUE)
 			if("Cancel", null)
 				return
+		if(credit)
+			to_chat(world, span_boldannounce(credit), confidential = TRUE)
```

Comment:
```suggestion
		if(credit)
			to_chat(world, span_announce(credit), confidential = TRUE)
```

i forget if `span_announce()` is a real span macro or not but I think it would look nicer if you didn't keep it the same span as the rest of the announcement. if you disagree please post photographs of what it currently looks like, and then with the change (or an alternative, better looking change) so I can visualize it better

Reply:
I'll do a quick test of this to see how it looks, if i think this looks fine ill do your suggestion but im always open for feedback really.

Reply:
ehhh after testing this all it does is make it not show up in any tab that doesnt have Info messages displayed, where as the main play internet sound text displays in every tab regardless of what is selected. As well as it functionally looks no different. 

![dreamseeker_dPe31Zs4mr](https://github.com/tgstation/tgstation/assets/2568378/51b6d44b-50d7-4697-a1bb-cc850029ccad)